pl2cpp2.doc

```diff
@@ -1717,13 +1719,16 @@ PREDICATE(). Their exception handling is as follows:
       calls).
 \end{description}
 
-\subsubsection{Sample PlBlob code}
+\subsubsection{Sample PlBlob code (connection to database)}
 \label{sec:cpp2-blobs-sample-code}
 
 Here is minimal sample code for creating a blob that owns a connection
 to a database. It has a single field (\exam{connection}) and
 defines compare_fields() and write_fields().
 
+A second sample code shows how to wrap a system pointer -
+\secref{cpp2-blobs-sample-code-wrap}
```

Comment:
This label does not exist.   Which one do you mean?

Reply:
Fixed with a force-push